examples/raytracer.cpp

```diff
@@ -43,7 +43,9 @@
 #include <cstring>
 
 extern "C"{
+#ifndef USE_MPI
```

Comment:
I do not understand this. If MPI is not available, force usage of MPI obviously cannot work. So what's the point?

Reply:
No idea. @twittidai? However, redefining a macro triggers a warning here, so I added an ```ifdef```